python/ml4ir/base/features/feature_config.py

```diff
@@ -18,6 +18,7 @@ class FeatureConfigKey:
     LABEL = "label"
     FEATURES = "features"
     RANK = "rank"
+    OLD_RANKING_SCORE = "s"
```

Comment:
@mohazahran You should definitely NOT be putting this in here.

Reply:
where should it be put then

Reply:
done